in_toto/runlib.py

```diff
@@ -119,7 +120,7 @@ def record_artifacts_as_dict(artifacts):
             the link command.
 
   <Exceptions>
-    None.
+    SettingsError if ARTIFACT_BASE_PATH or ARTIFACT_EXCLUDES can't be used
```

Comment:
It might be better if this listed the exception name: `in-toto.exceptions.SettingsError`, since the caller will have to specify the full name to correctly catch the exception.